src/app/shell/personal-cabinet/provider/create-workshop/create-description-form/create-description-form.component.ts

```diff
@@ -35,27 +36,28 @@ export class CreateDescriptionFormComponent implements OnInit, OnDestroy {
     this.DescriptionFormGroup = this.formBuilder.group({
       imageFiles: new FormControl(''),
       description: new FormControl('', [
-        Validators.required, 
+        Validators.required,
         Validators.minLength(ValidationConstants.INPUT_LENGTH_3),
-        Validators.maxLength(ValidationConstants.MAX_DESCRIPTION_LENGTH_500)
+        Validators.maxLength(ValidationConstants.MAX_DESCRIPTION_LENGTH_500),
       ]),
-      disabilityOptionsDesc: new FormControl({ value: '', disabled: true },[
+      disabilityOptionsDesc: new FormControl({ value: '', disabled: true }, [
         Validators.minLength(ValidationConstants.INPUT_LENGTH_1),
-        Validators.maxLength(ValidationConstants.INPUT_LENGTH_256) 
+        Validators.maxLength(ValidationConstants.INPUT_LENGTH_256),
       ]),
       keyWords: new FormControl(''),
       categories: this.formBuilder.group({
         directionId: new FormControl('', Validators.required),
         departmentId: new FormControl('', Validators.required),
         classId: new FormControl('', Validators.required),
-      })
+      }),
+      sectionItemsFormArray: this.SectionItemsFormArray,
```

Comment:
comment above regarding FormCotrol naming in the formGroup 
back-end Name === front-end Modle name === FormControl in the FormGroup naming

so, when you will patch value in edit mode, we will accept back naming, find the corresponding naming in the model and patch the corresponding formcontrol. In other case you need to manage this process by yourself